CrazyCanvas/Source/States/MultiplayerState.cpp

```diff
@@ -27,17 +27,17 @@ void LobbyState::Init()
 	RenderSystem::GetInstance().SetRenderStageSleeping("SKYBOX_PASS", true);
 	RenderSystem::GetInstance().SetRenderStageSleeping("SHADING_PASS", true);
 
-	m_LobbyGUI = *new LobbyGUI("Lobby.xaml");
-	m_View = Noesis::GUI::CreateView(m_LobbyGUI);
+	m_MultiplayerGUI = *new MultiplayerGUI("Lobby.xaml");
```

Comment:
Please rename the Lobby.xaml as well :)
```suggestion
	m_MultiplayerGUI = *new MultiplayerGUI("Multiplayer.xaml");
```

Reply:
oh- Right!